jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_method.cpp

```diff
@@ -290,6 +290,45 @@ SCENARIO(
   }
 }
 
+SCENARIO(
+  "java_bytecode_convert_method_with_getstatic",
+  "[core][java_bytecode][java_bytecode_convert_method]")
+{
+  GIVEN("A class that reads a static field.")
+  {
+    const symbol_tablet symbol_table = load_java_class(
+      "ClassReadingStaticField",
+      "./java_bytecode/java_bytecode_convert_method");
+
+    WHEN("Converting the method that reads said field")
+    {
+      const auto &method_symbol =
```

Comment:
:pick: if the symbol isn't found, using lookup_ref makes the output on CI not very clear, better to use lookup and then REQUIRE(method_symbol != nullptr)